docs/writing-tests/test-addon.mdx

```diff
@@ -190,7 +196,39 @@ When the addon is set up automatically, it will create or adjust your Vitest con
 
 ## Usage
 
-There are three primary ways to run tests using the Vitest plugin:
+There are multiple ways to run tests using the addon:
+
+### Storybook UI
+
+The easiest way to run tests is through the Storybook UI. To run all tests for your components, press the Run tests button in the test module at the bottom of the sidebar.
+
+<Video src="../_assets/writing-tests/addon-test-run-all-tests.mp4" />
+
+Alternatively, you can expand the test module to run specific types of tests individually. For those test types which have a watch mode (which will automatically re-run relevant tests upon code changes), you can toggle that on or off.
+
+![Screenshot of test module, expanded, showing test types and watch mode toggle](../_assets/writing-tests/addon-test-module-expanded.png)
+
+<Callout variant="info" icon="ℹ️">
+
+If you have the [Visual tests addon](./visual-testing.mdx) installed, you'll see an option to run Visual tests alongside [Component tests](./component-testing.mdx).
+
+![Screenshot of test module, expanded, showing Visual tests](../_assets/writing-tests/addon-test-module-expanded-with-vta.png)
+
+Other addons may also provide test types that can be run from the test module and affect the status indicators on stories and components.
+
+</Callout>
+
+After running your tests, you will now see status indicators on stories and components for their pass, fail, or error state. You can press on these indicators to see more details and jump straight to the failure in the Component tests addon panel. That panel provides an interactive debugger for your component test, allowing you to step through each simulated behavior or assertion.
+
+<Callout variant="info" icon="ℹ️">
+
+The Component tests addon panel replaces the [Interactions addon panel](../essentials/interactions.mdx) when the Test addon is installed. While the testing mechanism is different, the functionality of the addon panel itself remains the same.
+
+</Callout>
+
+The test module will also show you the total number of tests run, the number of tests that passed, and the number of tests that failed or errored. You can press the failure number to filter the sidebar to only those stories that failed.
+
+<Video src="../_assets/writing-tests/addon-test-filter-sidebar.mp4" />
```

Comment:
I believe this should be reframed and included inside a section on previewing the test results to educate the user on how to do it. There's a lot of information for the user to digest, which can result in confusion. The bonus is that we can use said section in the future for other learning materials, blogs, etc.

Reply:
I disagree. The section first describes how to run tests and then how to review them. One must come after the other. Splitting that into two sub-sections would be overhead.

Reply:
I'm going to go ahead and merge this into the [parent PR](https://github.com/storybookjs/storybook/pull/29241) so that it goes out with the actual feature.

If you really feel strongly about this feedback (or anything else), please ping me in a comment over there.